packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -385,10 +394,14 @@ describe("Odsp Error", () => {
 		assert.strictEqual(
 			error.errorType,
 			OdspErrorTypes.fileNotFoundOrAccessDeniedError,
-			"Error type should be locationRedirection",
+			"Error type should be fileNotFoundOrAccessDeniedError",
```

Comment:
? Doesn't this mean that the test was failing before (how?) or will fail after this change?

Reply:
This is strictly changing the error message we log, the actual check is unchanged - the message referenced the wrong kind of error (likely a copy-paste error).

Reply:
Ah, yes, I misread this 👍